src/transcript.zig

```diff
@@ -198,6 +198,9 @@ fn TranscriptT(comptime Hash: type) type {
         handshake_secret: ?[Hmac.mac_length]u8 = null,
         server_finished_key: [Hmac.key_length]u8 = undefined,
         client_finished_key: [Hmac.key_length]u8 = undefined,
+        binder: [Hmac.mac_length]u8 =undefined,
+        server_hmac: [Hmac.mac_length]u8 = undefined,
+        client_hmac: [Hmac.mac_length]u8 = undefined,
```

Comment:
I'm not deep enough into TLS in order to know if some of these variables could be shared cause the scope of them is short lived. Please let me know if this needs to be changed.

Reply:
Yes they are all short lived. They are always copied (@memcpy) to some writer.
I replaced those 3 three with one common buffer. 
Thanks for noticing this.